localization/english/culture/ctp_titles_l_english.yml

```diff
@@ -650,16 +653,70 @@
  king_muslim_male_russian:0 "Sultan"
  king_muslim_female_russian:0 "Sultansha"
  kingdom_muslim_russian:0 "Sultanat"
- count_male_russian_independent:0 "Knyaz'"
- count_female_russian_independent:0 "Knyaginya"
- county_russian_independent:0 "Knyazhestvo"
- duke_male_russian_independent:0 "Knyaz'"
- duke_female_russian_independent:0 "Knyaginya"
- duchy_russian_independent:0 "Knyazhestvo"
  count_tribal_male_russian:0 "Vozh"
  count_tribal_female_russian:0 "Vozh"
  county_tribal_russian:0 "Plemya"
 
+## Old East Slavic cultures
+ 
+ baron_male_ilmenian:0 "Zemĭskojĭ bojarine"
```

Comment:
Would Bojarine be capitalized?

Reply:
Of course also extends to the other barons

Reply:
The practice of capitalizing every word in a title looks markedly un-Russian. That's a decision to be made whether we want cultural or cross-cultural consistency in this respect.
(Though note that at least German should probably have capitalization in such cases because these are nouns.)